core/src/dht/actions/hold.rs

```diff
@@ -37,8 +34,6 @@ pub struct HoldEntryFuture {
     address: Address,
 }
 
-impl Unpin for HoldEntryFuture {}
```

Comment:
@zippy why is this removed? Assuming the compiler thinks it's no longer necessary?

Reply:
I had to.  Apparently Unpin became a private element (the use statement broke and I had to remove it), and gets auto-implemented in the default case.  I didn't dig too deep, just got it to work.